_minutes/2025-04-10-wecg.md

```diff
@@ -0,0 +1,177 @@
+Call-in details: [WebExtensions CG, 10th April 2025](https://www.w3.org/events/meetings/0090c842-271b-4194-b93e-9d401d07af5e/20250410T080000/)
+Zoom issues? Ping @zombie (Tomislav Jovanovic) in [chat](https://github.com/w3c/webextensions/blob/main/CONTRIBUTING.md#joining-chat)
+
+
+## Agenda: [discussion in #797](https://github.com/w3c/webextensions/issues/797), [github issues](https://github.com/w3c/webextensions/issues)
+
+The meeting will start at 3 minutes after the hour.
+
+See [issue 531](https://github.com/w3c/webextensions/issues/531) for an explanation of this agenda format.
+
+ * **Announcements** (5 minutes)
+   * Recap Berlin F2F
+ * **Triage** (15 minutes)
+   * [Issue 795](https://github.com/w3c/webextensions/issues/795): Proposal: opt-out to transformations of extension icons
+   * [Issue 800](https://github.com/w3c/webextensions/issues/800): Proposal: browser.runtime.getDocumentId()
+   * [Issue 803](https://github.com/w3c/webextensions/issues/803): Proposal: Added Sync management APIs
+   * [Issue 805](https://github.com/w3c/webextensions/issues/805): Proposal: Add once as an option in API addListener
+   * [Issue 806](https://github.com/w3c/webextensions/issues/806): Proposal: extension specific CSS env() variables
+   * [Issue 807](https://github.com/w3c/webextensions/issues/807): Proposal: add OPTIONS_UI as ContextType
+ * **Timely issues** (10 minutes)
+   * [Issue 698](https://github.com/w3c/webextensions/issues/698): MessageFormat 2 support
+ * **Check-in on existing issues** (15 minutes)
+   * [PR 791](https://github.com/w3c/webextensions/pull/791): Add trial_tokens key to specification
+   * [PR 792](https://github.com/w3c/webextensions/pull/792): Proposal: runtime.onInvalidated event
+   * [PR 798](https://github.com/w3c/webextensions/pull/798): WIP: Proposal "Allow to change permissions behavior on extension's update"
+   * [PR 799](https://github.com/w3c/webextensions/pull/799): Add browser.permissions.canAccess() proposal
+   * [PR 802](https://github.com/w3c/webextensions/pull/802): Add proposal for topFrameMatches and excludeTopFrameMatches
+
+
+## Attendees (sign yourself in)
+
+ 1. Rob Wu (Mozilla)
+ 2. Giorgio Maone (NoScript, Tor)
+ 3. Timothy Hatcher (Apple)
+ 4. Simeon Vincent (Mozilla)
+ 5. Oliver Dunk (Google)
+ 6. Tomislav Jovanovic (Mozilla)
+ 7. Aaron Selya (Google)
+ 8. David Johnson (Apple)
+ 9. Eemeli Aro (Mozilla L10n)
+ 10. Carlos Jeurissen (Jeurissen Apps)
+ 11. Casey Garland (Capital One)
+ 12. Mukul Purohit (Microsoft)
+ 13. Maxim Topciu (AdGuard)
+
+
+## Meeting notes
+
+Recap Berlin F2F
+
+ * [timothy] 2 weeks ago we met face-to-face. Four fully packed days, including issue triage and in depth discussions. DNR, cosmetic rules, future direction, etc. Looking forward to the next face-to-face. Plan is to have it be later than usual because we try to meet every 6 months or so, and the next TPAC is later than usual this year (November). Probably closer to April 2025.
```

Comment:
```suggestion
 * [timothy] 2 weeks ago we met face-to-face. Four fully packed days, including issue triage and in depth discussions. DNR, cosmetic rules, future direction, etc. Looking forward to the next face-to-face. Plan is to have it be later than usual because we try to meet every 6 months or so, and the next TPAC is later than usual this year (November). Probably closer to April 2026.
```